File: navigation/src/main/java/com/example/navigation/stepsEngine/field/TaxSelectionField.java
package com.example.navigation.stepsEngine.field;

import com.example.navigation.stepsEngine.payment.FlowState;

public class TaxSelectionField extends Field {

    @Override
    public String get(FlowState flowState) {
        return flowState.getTaxId();
    }

    @Override
    public String getId() {
        return null;
    }

    @Override
    public void set(FlowState flowState, Object value) {

    }

}

File: navigation/src/main/java/com/example/navigation/stepsEngine/field/TaxSelectionField.kt
package com.example.navigation.stepsEngine.field

import com.example.navigation.stepsEngine.payment.FlowState
import kotlinx.android.parcel.Parcelize

@Parcelize
class TaxSelectionField : Field() {
    override operator fun get(flowState: FlowState): String? {
        return flowState.taxId
    }

    override fun getId(): String {
        return FieldName.TAX
    }

    override operator fun set(flowState: FlowState, value: Any?) {}
}